Update announcements.json

TamGIBill opened the pull request:
Added Webinar Slides Are Available for the VBA Education March SCO/SAA Office Hours - COVID-19 Announcement

## Description


## Testing done


## Screenshots


## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

TamGIBill committed:
Update announcements.json

Added Webinar Slides Are Available for the VBA Education March SCO/SAA Office Hours - COVID-19 Announcement

shawkey-daniel commented on src/applications/static-pages/school-resources/constants/announcements.json:
```diff
@@ -1,4 +1,11 @@
+    "url": "https://www.benefits.va.gov/gibill/#20200330",
+    "date": "2020-03-31",
+    "displayStartDate": "2020-03-31",
+    "displayEndDate": "2020-04-31"
```
There is no such thing as April 31st :)

jhonnyoddball replied:
oh yeah, nice catch

TamGIBill committed:
Update announcements.json

Updated 4/31 date and added April Webinars - 107 Recertification Training Announcements

shawkey-daniel reviewed (approved):
LGTM

TamGIBill commented:
DOH!

Tammy S. Hurley
Education Service
Veterans Benefits Administration
202-461-9838

[bye3]


From: shawkey-daniel <notifications@github.com>
Sent: Tuesday, March 31, 2020 9:20 AM
To: department-of-veterans-affairs/vets-website <vets-website@noreply.github.com>
Cc: Hurley, Tammy, VBAVACO <tammy.hurley1@va.gov>; Author <author@noreply.github.com>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/vets-website] Update announcements.json (#12057)


@shawkey-daniel requested changes on this pull request.

________________________________

In src/applications/static-pages/school-resources/constants/announcements.json<https://github.com/department-of-veterans-affairs/vets-website/pull/12057#discussion_r400906654>:

> @@ -1,4 +1,11 @@

 [

+  {

+    "name": "Webinar Slides Are Available for the VBA Education March SCO/SAA Office Hours - COVID-19",

+    "url": "https://www.benefits.va.gov/gibill/#20200330",

+    "date": "2020-03-31",

+    "displayStartDate": "2020-03-31",

+    "displayEndDate": "2020-04-31"

There is no such thing as April 31st :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/department-of-veterans-affairs/vets-website/pull/12057#pullrequestreview-384723566>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AL5DFAHX46JFVO545OR4E5TRKHUYPANCNFSM4LXREUVQ>.
